Implement env-list command

fsouza opened the pull request:
It just loads the configuration file and displays its content in a
table.

aflag commented on testdata/.tranor/envs.json:
```diff
@@ -0,0 +1,14 @@
+[
+	{
+		"name": "dev",
+		"dnsSuffix": "dev.mycompany.com"
```
I suggest changing mycompany.com to example.com as it doesn't make it less understandable and it does not use a valid DNS name.

aflag commented on testdata/.tranor/envs.json:
```diff
@@ -0,0 +1,14 @@
+		"name": "prd",
+		"dnsSuffix": "mycompany.com"
+	}
+]
```
We are missing QA environment. Also, why not have the full name development, staging, production instead of 3 letter acronyms? Specially prd seems very strange :P

tatiana replied:
I don't think full names are the best solutions - but I agree `stg` and `prd` look weird. I suggest we keep the current naming:

```
- dev
- qa
- stage
- prod
```

aflag replied:
Why not use the full name? What do we lose?

fsouza replied:
This is a test file. The names aren't actually relevant. But I updated the code to use dev, qa, stage and production.

aflag commented on testdata/.tranor/envs.json:
```diff
@@ -0,0 +1,14 @@
+[
```
Is this testdata/.tranor what will be installed on the user's home by default when installing tranor? Shouldn't it be defined somewhere else with, perhaps, a link to it on testdata?

tatiana replied:
@aflag I believe this is just a test file, not the default for installation. @fsouza: could you confirm please

aflag replied:
I guess you are right. I thought there would be a default configuration file installed without EF stuff, but I don't know if that makes sense.

fsouza replied:
This is indeed just for test. This change doesn't cover the setup process yet. See my comment below for some ideas on how to install it.

aflag commented:
Good job, @fsouza ! I had a few minor comments before approval.

tatiana commented:
@fsouza as tranor is open source, it won't make sense to add EF's .tranor configs here - but we need to find a way of making the building process easy for the company. How were you thinking to deal with this? (BTW this can be discussed in Jira, but I strongly believe it is part of this task)

Also, any special reason for organizing the configuration in a directory (`.tranor`) and having JSON's per functionality (eg. `envs`) - and not having a single configuration file?

Commit:
Implement env-list command

It just loads the configuration file and displays its content in a
table.

fsouza commented:
@tatiana this would be part of tranor's install process. We could make the setup with internal information part of a private repository, or make it more similar to tsuru:
- user setups a target
- user runs `tranor setup` to download tranor and tsuru configuration from this target

And the target can be a static application on tsuru that serves the configuration.

Regarding the file name and location, I don't think we will need any other configuration file, but I understand what you mean. I updated the format to be a single config file and made "envs" one of the configuration options, I also renamed the file path to be `$HOME/.tranor/config.json`.

I can renamed it to `$HOME/.tranor-config.json` as well, if that's preferred. No strong feelings here.

tatiana commented:
@fsouza: :+1: for `$HOME/.tranor/config.json` and the install procedure
